services/ui-src/src/utils/other/mask.test.ts

```diff
@@ -71,6 +83,29 @@ describe("Test convertToCommaSeparatedString", () => {
   });
 });
 
+describe("Test convertToCommaSeparatedRatioString", () => {
+  test("Valid ratio is split and masked on each side individually", () => {
+    for (let testCase of ratioMaskAcceptableTestCases) {
+      expect(convertToCommaSeparatedRatioString(testCase.test)).toEqual(
+        testCase.expected
+      );
+    }
+  });
+  test("Check if non number strings can fail gracefully", () => {
+    const testCases = [
+      { test: "No numbers here me lord", expected: ":" },
```

Comment:
If we're gonna go here, might as well go all in. 😉 
```suggestion
      { test: "No numbers here, m'lord", expected: ":" },
```

Reply:
Fixed :)